modules/tracker/models/base/SubmissionModelBase.php

```diff
@@ -90,4 +90,28 @@ public function __construct()
      * @return array submission DAOs
      */
     abstract public function getScalars($submissionDao, $key = false);
+
+    /**
+     * Get the single latest submission associated with a given producer.
+     *
+     * @param Tracker_ProducerDao $producerDao producer DAO
+     * @param false | string $date the end of the interval or false to use 23:59:59 of the current day
+     * @param string $branch the branch of the submission for which to search
+     * @param bool $onlyOneDay if true return submissions 24 hours back from $date, false otherwise. In the case of
```

Comment:
Last comment.  I didn't notice this before, but `false otherwise` seems strange here, also adding "searching a maximum of 24 hours", see below.

Maybe

`$onlyOneDay if true return submissions searching a maximum of 24 hours back from $date, otherwise unlimited search into the past.`

even better if this can be said in fewer words